From amluto@gmail.com  Tue Apr 28 15:45:16 2009
From: Andrew Lutomirski <amluto@gmail.com>
Date: Tue, 28 Apr 2009 15:53:02 -0400
Subject: USB: Fix makefile so that CONFIG_WDM and CONFIG_TMC work.
To: gregkh@suse.de
Message-ID: <cb0375e10904281253y67349baaq6588a3933720f89b@mail.gmail.com>


From: Andy Lutomirski <amluto@gmail.com>

If CONFIG_USB_ACM and CONFIG_USB_PRINTER are not set, then
cdc-wdm and usbtmc won't get built.

Signed-off-by: Andy Lutomirski <amluto@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/Makefile |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/usb/Makefile
+++ b/drivers/usb/Makefile
@@ -27,6 +27,8 @@ obj-$(CONFIG_USB_WUSB)		+= wusbcore/
 
 obj-$(CONFIG_USB_ACM)		+= class/
 obj-$(CONFIG_USB_PRINTER)	+= class/
+obj-$(CONFIG_USB_WDM)		+= class/
+obj-$(CONFIG_USB_TMC)		+= class/
 
 obj-$(CONFIG_USB_STORAGE)	+= storage/
 obj-$(CONFIG_USB)		+= storage/
